src/fswAlgorithms/orbitControl/lambertValidator/lambertValidator.rst

```diff
@@ -0,0 +1,106 @@
+indicate that the Lambert solution is not valid or has not converged yet.
+
+
+Message Connection Descriptions
+-------------------------------
+The following table lists all the module input and output messages.
+The module msg connection is set by the user from python.
+The msg type contains a link to the message structure definition, while the description
+provides information on what this message is used for.
+
+.. list-table:: Module I/O Messages
+    :widths: 25 25 50
+    :header-rows: 1
+
+    * - Msg Variable Name
+      - Msg Type
+      - Description
+    * - navTransInMsg
+      - :ref:`NavTransMsgPayload`
+      - translational navigation input message
+    * - lambertProblemInMsg
+      - :ref:`lambertProblemMsgPayload`
+      - lambert problem setup input message
+    * - lambertSolutionInMsg
+      - :ref:`LambertSolutionMsgPayload`
+      - lambert problem solution input message
+    * - lambertPerformanceInMsg
+      - :ref:`LambertPerformanceMsgPayload`
+      - lambert problem performance message (additional information about the solution process)
+    * - dvBurnCmdOutMsg
+      - :ref:`DvBurnCmdMsgPayload`
+      - Delta-V command output message
+
+
+Module Assumptions and Limitations
+----------------------------------
+The equations of motion used inside the module to propagate the state assume simple two body point mass gravity, and the
+motion is propagated using a 4th order Runge-Kutta (RK4). Additionally, this module assumes that
+:math:`t_{final} > t_{maneuver} > t`, with final time :math:`t_{final}`, maneuver time :math:`t_{maneuver}` and
+current time :math:`t`.
+
+
+Algorithm
+---------
+Equations of motion (two body point mass gravity) with gravitational parameter :math:`\mu` and spacecraft position
+vector :math:`\mathbf{r}`:
+
+.. math::
+    :label: eq:EOM
+
+    \mathbf{\ddot{r}} = - \frac{\mu}{r^3} \mathbf{r}
+
+The 27 perturbed initial states that are propagated to check for any constraint violations are obtained in the
+following way:
+The uncertainty of each state, specified by the 6x6 matrix "uncertaintyStates" in Hill frame components, is added to
+each state in the plus and minus direction.
+For :math:`N=6` states, this gives :math:`2 \times N = 12` initial states.
+The uncertainty of the Delta-V magnitude is applied to the Delta-V vector for each of those initial states, both in the
+Delta-V direction and against the Delta-V direction (corresponding to maximum and minimum expected DV magnitude).
+Applied to the 12 initial states, this gives 24 states.
+The last 3 initial states are obtained by applying the maximum and minimum expected DV to the unperturbed spacecraft
+state (+2), and by leaving the state and Delta-V vector entirely unperturbed (+1).
+
+
+User Guide
+----------
+The module is first initialized as follows:
+
+.. code-block:: python
+
+    module = lambertValidator.LambertValidator()
+    module.ModelTag = "lambertValidator"
+    module.finalTime = 2000.
```

Comment:
update user guide to use setter method

Reply:
Added setters and getters for all module variables.